CHANGELOG.md

```diff
@@ -5,11 +5,19 @@ It was first called Evopedia (and was using the file format of Evopedia).
 Then it was renamed Kiwix-html5 (and uses ZIM files), and then again was renamed to Kiwix-JS.
 
 ## Kiwix-JS v2.8.0
+
 Released on *TODO*
```

Comment:
Presumably, put correct date in before merging just before we're ready to create tag.

Reply:
That's right

Reply:
That can probably be updated with today, now

Reply:
OK will do it now and merge.